hudi-spark2/src/main/java/org/apache/hudi/internal/DefaultSource.java

```diff
@@ -18,7 +18,7 @@
 
 package org.apache.hudi.internal;
 
-import org.apache.hudi.DataSourceUtils;
+import org.apache.hudi.DataSourceUtilsForSpark2;
```

Comment:
Because ```hudi-spark``` depends on ```hudi-spark2```. I cannot also let ```hudi-spark2``` depends on ```hudi-spark``` tho copying files is not a clean way.